### ISSUE TITLE
2.5 Input Modalities

### ISSUE THREAD

**Issue by DonnaDuiker** [alkem-io/client-web#8373]
# Background
After going through the WCAG 2.1 checklist, creating issues for all A and AA points that still need work. 1 issue for each guideline.

This is guideline 2.5 Input Modalities

# Acceptance Criteria

- [ ] 2.5.3 Label in Name: Visible labels match accessible names (for screen readers).	(need to be checked: are label same as label name? If label is missing this is also not working)
- [ ] 2.5.5 Target Size: Touch targets are at least 44x44 CSS pixels (or spaced adequately). (Bacause of the breadcrumbs and all icons(maybe more?))

# Context

[Excel file](https://stichtingcherrytwist.sharepoint.com/:x:/s/Product/EawxIsWzBaRHt8K2E1tUwmsBzd93mWRRiImOP2M0muxujQ?e=eYV2Qr)

[W3C website](https://www.w3.org/TR/WCAG21/)
